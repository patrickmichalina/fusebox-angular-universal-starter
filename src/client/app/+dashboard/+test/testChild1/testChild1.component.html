<md-card>
  Test Child 1 Project
</md-card>