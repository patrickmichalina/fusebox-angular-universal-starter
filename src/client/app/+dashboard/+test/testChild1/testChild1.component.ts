import { ChangeDetectionStrategy, Component } from '@angular/core'

@Component({
  selector: 'pm-test-child-2',
  templateUrl: './testChild1.component.html',
  styleUrls: ['./testChild1.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class TestChild1Component {
}
