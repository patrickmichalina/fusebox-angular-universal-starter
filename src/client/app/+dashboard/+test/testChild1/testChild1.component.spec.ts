import { async, ComponentFixture, TestBed } from '@angular/core/testing'
import { Component } from '@angular/core'
import { TestModule } from '../test.module'
import { TestChild1Component } from './testChild1.component'

describe(TestChild1Component.name, () => {
  let fixture: ComponentFixture<TestChild1Component>

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [TestModule],
      declarations: [TestComponent]
    }).compileComponents()
  }))

  beforeEach(async(() => {
    fixture = TestBed.createComponent(TestChild1Component)
  }))

  afterEach(async(() => {
    TestBed.resetTestingModule()
  }))

  it('should match snapshot', async(() => {
    expect(fixture).toMatchSnapshot()
  }))

  it('should compile', async(() => {
    expect(fixture.nativeElement).toBeDefined()
  }))
})

@Component({
  selector: 'test-component',
  template: '<pm-test-child-2></pm-test-child-2>'
})
class TestComponent {}
