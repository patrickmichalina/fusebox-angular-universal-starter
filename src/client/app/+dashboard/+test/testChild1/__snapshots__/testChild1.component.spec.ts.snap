// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestChild1Component should match snapshot 1`] = `
<pm-test-child-2>
  <md-card
    class="mat-card"
  >
    
    Test Child 1 Project
  
  </md-card>
</pm-test-child-2>
`;
